canvas_modules/common-canvas/src/common-properties/common-properties.jsx

```diff
@@ -252,7 +253,8 @@ CommonProperties.propTypes = {
 		convertValueDataTypes: PropTypes.bool,
 		showRequiredIndicator: PropTypes.bool,
 		showAlertsTab: PropTypes.bool,
-		locale: PropTypes.string
+		locale: PropTypes.string,
+		iconSwitch: PropTypes.bool
```

Comment:
Why do we need this?  If they are passing in the `propertyIconHandler` they shouldn't also need to set this config option.